### PR TITLE
Feature/twitter text cleaner

### PR DESCRIPTION
**Pull request by lakshyauttrani**
this code takes the tweet data and cleans it to give a clean tweet along with removing the stopwords

### CODE REVIEW

File: Tweet Cleaner/Twitter_Text_Cleaner.ipynb [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        "id": "0pzOePX8UMfq"
+      },
+      "source": [
+        "from nltk.tokenize import WordPunctTokenizer\r\n",
+        "tok = WordPunctTokenizer()\r\n",
+        "pat1 = r'@[A-Za-z0-9]+'\r\n",
+        "pat2 = r'https?://[A-Za-z0-9./]+'\r\n",
+        "combined_pat = r'|'.join((pat1, pat2))\r\n",
+        "def tweet_cleaner(text):\r\n",
+        "    soup = BeautifulSoup(text, 'lxml')\r\n",
```
Comment (jediXNavi) [inline Tweet Cleaner/Twitter_Text_Cleaner.ipynb]: Why do we need BeautifulSoup to parse the text? that's anyway stored in a text column na! Why do we parse it to an xml format? Why can't we read directly and do pd.apply?

File: Tweet Cleaner/Twitter_Text_Cleaner.ipynb [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "Eiy5GYwDWJ9_",
+        "outputId": "47ff835b-1322-4533-9e7b-4c322e326559"
+      },
+      "source": [
+        "test_result = []\r\n",
```
Comment (jediXNavi) [inline Tweet Cleaner/Twitter_Text_Cleaner.ipynb]: I think let's not have a list here. If the data we want( talking from a production standpoint) is in millions, the list would be infeasible. Instead let's directly apply a pandas function! For now, that should be fine. Incase we need better processing can switch over to some Big data framework.

File: Tweet Cleaner/Twitter_Text_Cleaner.ipynb [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+      },
+      "source": [
+        "tokenizer = nltk.RegexpTokenizer(r\"\\w+\")\r\n",
+        "\r\n",
+        "def special_character_cleaner(text):\r\n",
+        "  new_words = tokenizer.tokenize(text)\r\n",
+        "  return text\r\n",
+        "\r\n",
+        "def get_text_processing(text):\r\n",
+        "    stpword = stopwords.words('english')\r\n",
```
Comment (jediXNavi) [inline Tweet Cleaner/Twitter_Text_Cleaner.ipynb]: For every tweet text, it will get the list of stopwords. Better to have it as a global variable

**Review by jediXNavi — CHANGES_REQUESTED** [review]

File: Tweet Cleaner/Twitter_Text_Cleaner.ipynb (hunk 1 above)
Comment (lakshyauttrani) [reply Tweet Cleaner/Twitter_Text_Cleaner.ipynb]: @jediXNavi  would pd.apply be able to identify any text that has lxml properties??

File: Tweet Cleaner/Twitter_Text_Cleaner.ipynb (hunk 3 above)
Comment (lakshyauttrani) [reply Tweet Cleaner/Twitter_Text_Cleaner.ipynb]: @jediXNavi makes complete sense will update this bit and recommit the code

**Review by jediXNavi — APPROVED** [review]

**Pull request merged**